content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -1,130 +1,135 @@
-[among others](/doc/command-reference/remote/add#supported-storage-types)).
-
-> For hands-on experience, we recommend following the
-> [versioning tutorial](/doc/use-cases/versioning-data-and-model-files).
-
-## DVC is not Git!
-
-DVC metafiles such as `dvc.yaml` and `.dvc` files serve as placeholders to track
-data files and directories for versioning (among other purposes). They point to
-specific data contents in the <abbr>cache</abbr>, providing the ability to store
-multiple data versions out-of-the-box.
-
-Full-fledged
-[version control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control)
-is left for Git and its hosting platforms (e.g. GitHub, GitLab) to handle. These
-are designed for source code management (SCM) however, and thus ill-equipped to
-support data science needs. That's where DVC comes in: with its built-in data
-<abbr>cache</abbr>, reproducible [pipelines](/doc/start/data-pipelines), among
-several other novel features (see [Get Started](/doc/start/) for a primer.)
-
-## Track data and models for versioning
+# Versioning Data and Models
+
+Data science teams today face data management questions around versioning
+datasets, data artifacts, and machine learning models. How do we keep track of
+changes in data, code, and ML models? What’s the best way to organize and store
+multiple versions of data files for safe, persistent access? How can the
+lifecycle of data and models be followed and enforced?
+
+![](/img/data_ver_complex.png) _Exponential complexity of DS projects_
+
+DVC proposes to _codify_ data projects in order to adopt existing engineering
+tools like Git
+[version control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control),
+continuous integration (CI), and other best practices that improve productivity.
+This means writing a description of which data, ML artifacts, etc. should be in
+the environment at a given time. DVC can restore the <abbr>workspace</abbr>
+files and directories (from a separate data storage) to match that description.
+Here are some benefits of this approach:
+
+- Track all the things (code, data, ML models) with a unified toolset that is
+  accessible to every project participant (data scientists, engineers, managers,
+  etc.).
+- Reproducibility and trustworthiness: identify exact research inputs, enable
+  anyone to understand and verify results.
+- Enforce data lifecycle policies — all project changes have to go through the
+  repository history.
+- Low coupling: separate code from data by caching large files/directories
+  automatically. This makes your project easier to maintain (high cohesion) and
+  improves data persistence.
+- Simple CLI: work with simple terminal [commands](/doc/command-reference)
+  similar to `git`.
+- Treating data as code also allows for other advanced features, see
+  [Get Started](/doc/start) for a primer.
+- Security: Audit an immutable history of changes to your data and models.
```

Comment:
Most of these benefits are about data versioning in general but I don't want to add another motivation paragraph or have 2 separate bullet lists.

Reply:
This list is already kind of long...

Reply:
> Simple CLI...

The purpose of this bullet is to also mention the Viewer and VSCode plugin when they are released. Otherwise I would merge it with the first one about the "unified toolset".

Reply:
p.s. I did some updates to this list.